pac4j-jwt/src/main/java/org/pac4j/jwt/credentials/authenticator/JwtAuthenticator.java

```diff
@@ -70,7 +84,17 @@ private void warning() {
      * @since 1.8.2
      */
     public JwtAuthenticator(final String signingSecret, final String encryptionSecret) {
-        this.signingSecret = signingSecret;
+    	setSigningSecret(signingSecret);
```

Comment:
Yes. If the string is a pem you need to specify which algorithm which will create the right Key object. In addition you can create the key object of the right type yourself